### DISCUSSION TITLE
Questions about accBias

### DISCUSSION THREAD

**Discussion by Namesume** [groops-devs/groops#295] [Q&A] [answered]
Hello developers,

I'm currently trying to recover garvity-field models by "scenarioGraceGravityfieldRecovery". In the step "020groopsVariational", I got two parameter files which recorded the accbias parameters, and I have two quesetions about it.

Queseiton 1, 
 I read the article "Beate Klinger, Torsten Mayer-Gürr,The role of accelerometer data calibration within GRACE gravity field recovery: Results from ITSG-Grace2016,Advances in Space Research,Volume 58, Issue 9, 2016", which explained the bias caculation method by using cubic basis splines, and I saw that the parameter files also correspond with the article (cubic spines, number of bias parameter and so on). May I ask if the article and scenario use the same calculation method? If so, why are there 7 sets of parameters instead of 4 sets (refer to Figure)? Shouldn't the 4 intervals within a day be calculated (knot interval length is set to 6h, and the 4 intervals are 00:00 to 06:00, 06:00 to 12:00, 12:00 to 18:00 and 18:00 to 00:00 respectively) ? 
![Fig 1](https://github.com/user-attachments/assets/8ebcf11d-fbea-44c4-828e-3c26ae993117)

Question 2,
Can accBias parameters get from "020groopsVariational" be used in accelerometer data calibration directly? If so, which set of parameters should I choose? If not, how can I calculate the correct accBias by GROOPS?

The article and GROOPS helps me a lot! Thank you in advance for any help you might provide. 

Sincerely.

Haonan Shi.

**Comment by tmayerguerr** [comment, marked as answer]
Hi Haonan Shi,

Question 1:
The article and scenario use the same calculation method. Cubic splines in 4 intervals needs 7 parameters, see the figure in [Basis splines](https://groops-devs.github.io/groops/html/fundamentals.basisSplines.html).

Question 2:
There is the program [InstrumentAccelerometerApplyEstimatedParameters](https://groops-devs.github.io/groops/html/InstrumentAccelerometerApplyEstimatedParameters.html). Use the same settings as the PreprocessingVariational... programs.

best regards
Torsten

**Reply by Namesume** [reply]
Thank you so much for your help!

**Comment by Namesume** [comment]
自动回复：您好，我已收到您的邮件。
